project/CakeDependencies.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
+// License: http://www.apache.org/licenses/LICENSE-2.0
+
+import sbt._
+
+object CakeDependencies {
+
+  // TODO: CO-68: remove JSR305 dependency when SBT moves away from Scala 2.10
+  val config = "com.typesafe" % "config" % "1.3.1"
+  val guava = "com.google.guava" % "guava" % "19.0"
+  val httpclient = "org.apache.httpcomponents" % "httpclient" % "4.3.6"
+  val jsr305 = "com.google.code.findbugs" % "jsr305" % "3.0.2"
+  val leveldbjni = "org.fusesource.leveldbjni" % "leveldbjni" % "1.7"
+  val quasiquotes = "org.scalamacros" %% "quasiquotes" % "2.1.0"
+  val slf4j = "org.slf4j" % "slf4j-api" % "1.7.12"
+
+  object Akka {
+    val version = "2.3.11"
+    val actor = "com.typesafe.akka" %% "akka-actor" % version
+  }
+
+  object Commons {
+    val codec = "commons-codec" % "commons-codec" % "1.6"
+    val comporess ="org.apache.commons" % "commons-compress" % "1.9"
+    val langs = "org.apache.commons" % "commons-lang3" % "3.4"
+    val logging = "commons-logging" % "commons-logging" % "1.1.3"
+  }
+
+  object Sbt {
```

Comment:
As far as I know we can't have access to objects defined in `project` folder (meta-build) in the main code (proper build) as we can't obtain the project name in the main code directly from `build.sbt`. If it's true, the same rule applies here; the context of `/project/project/CakePluginDependencies.scala` is different than the files in `/project`.